app/models/resource.rb

```diff
@@ -114,7 +114,7 @@ def search account: nil, kind: nil, owner: nil, offset: nil, limit: nil, search:
           raise ArgumentError, "'offset' contains an invalid value. 'offset' must be an integer greater than or equal to 0."
         end
 
-        scope = scope.order(:resource_id).limit(
```

Comment:
This is an excellent find, @john-odonnell!

However, I would suggest that the fix should be to make sure the `order` is applied to all queries, rather than remove it for all queries (I didn't see another order instance, but I also may have missed it).

It's important that the order be consistent, and if no order is given, [postgresql documents that there is no order guarantee](https://www.postgresql.org/docs/current/queries-order.html#:~:text=If%20sorting%20is%20not%20chosen,returned%20in%20an%20unspecified%20order.).

Reply:
Good catch, thanks. I updated the PR to order the resources before limiting based on `limit` & `offset`.

```rb
# Sort results alphabetically by resource ID
scope = scope.order(:resource_id)

if offset || limit
  ...
```